hotness/consumers.py

```diff
@@ -26,7 +26,9 @@
 from requests.packages.urllib3.util import retry
 import fedmsg
 import fedmsg.consumers
+import fedmsg.meta
 import requests
+import yaml
```

Comment:
Sorry I missed this first go around, could you please add this dependency to the requirements.txt?

Reply:
Oh, for sure.

Reply:
Done in 9c97fcf.